TACC/protx-dashboard
# hotfix/WG-update-python-client-and-packages

## Overview: ##
Updated poetry and packages to fix a docker build error


## Summary of Changes: ##
Updated poetry and some package dependencies 

## Testing Steps: ##
Try and build the containers to make sure you can successful build and run locally

## UI Photos:

## Notes: ##
